bin/bintray_garbage_collector.rb

```diff
@@ -0,0 +1,23 @@
+require 'dotenv'
+require './lib/garbage_collector.rb'
```

Comment:
```suggestion
require_relative '../lib/garbage_collector.rb'
```

otherwise it will try to reference relative to the current pwd (as opposed to the path relative to the current file)

Reply:
Nice catch! Thanks